Assets/Scripts/CoinGenerator.cs

```diff
@@ -0,0 +1,50 @@
+﻿using UnityEngine;
+using System.Collections.Generic;
+
+public class CoinGenerator : MonoBehaviour
+{
+    [SerializeField] private GameObject _coin;
```

Comment:
https://trello.com/c/orS9j3Rj/6-%D0%B4%D0%B8%D0%BD%D0%B0%D0%BC%D0%B8%D1%87%D0%B5%D1%81%D0%BA%D0%BE%D0%B5-%D1%81%D0%BE%D0%B7%D0%B4%D0%B0%D0%BD%D0%B8%D0%B5-%D0%BE%D0%B1%D1%8A%D0%B5%D0%BA%D1%82%D0%BE%D0%B2